GitUI/UserControls/RevisionGrid.cs

```diff
@@ -323,7 +324,7 @@ public void SetInitialRevision(GitRevision initialSelectedRevision)
             _initialSelectedRevision = initialSelectedRevision != null ? initialSelectedRevision.Guid : null;
         }
 
-        private bool _isRefreshingRevisions = false;
```

Comment:
Resharper is only a tool with a lack of empathy. There are many contributors without c# background. Declaring a field this way, they don't have to wonder how/if fields are initialized in c#. Changes like that bring no added value and distracts while reviewing.